ros2_graph/__main__.py

```diff
@@ -91,13 +92,28 @@ def get_style(style_file: str = None):
         ),
     }
 
+    to_ignore["nodes"] = to_ignore.get("nodes", ["/Graph_generator"])
+
+    to_ignore["topics"] = to_ignore.get(
+        "topics",
+        [
+            "/parameter_events",
+            "/rosout",
+            "/tf",
+            "/cascade_lifecycle_activations",
+            "/cascade_lifecycle_states",
+            "",
+        ],
```

Comment:
I think those should be always ignored right? Or do you see any situation where this wouldn't be true? So if the user forgets to add those they would be displayed
If that's the case, it would be better to add them after getting what's defined by the user. We should also use sets instead of lists since it is faster the search operations